pkg/cri/resource-manager/cache/cache.go

```diff
@@ -446,6 +455,26 @@ type Device struct {
 	Permissions string
 }
 
+// PageMigrate contains the policy/options for container page migration.
+type PageMigrate struct {
+	SourceNodes system.IDSet // idle memory pages on these NUMA nodes
+	TargetNodes system.IDSet // should be migrated to these NUMA nodes
+}
+
+func (pm *PageMigrate) Clone() *PageMigrate {
```

Comment:
nit: add a comment/docstring to the exported methods

Reply:
Added docstring.